DEVELOPMENT.md

```diff
@@ -54,3 +54,24 @@ development command line options.
   metadata cache we use only released portion of `dandi.__version__` as a token.
   If handling of metadata has changed while developing, set this env var to
   `clear` to have cache `clear()`ed before use.
+
+## Sourcegraph
+
+The [Sourcegraph](https://sourcegraph.com) browser extension can be used to
+view code coverage information as follows:
+
+1. Install the [Sourcegraph browser
+   extension](https://docs.sourcegraph.com/integration/browser_extension) in
+   your browser (Chrome or Firefox only)
+
+2. [Sign up](https://sourcegraph.com/sign-up) for a Sourcegraph account if you
+   don't already have one.  You must be signed in to Sourcegraph for the
+   remaining steps.
+
+3. Enable the [Codecov Sourcegraph
+   extension](https://sourcegraph.com/extensions/sourcegraph/codecov)
+
+4. On GitHub, when viewing a dandi-cli source file (either on a branch or in a
+   pull request diff), there will be a "Coverage: X%" button at the top of the
+   source listing.  Pressing this button will toggle highlighting of the source
+   lines based on whether they are covered by tests or not.
```

Comment:
Notes to myself ;):
AHA! FWIW in Chromium I didn't see Coverage button in diffs and highlighting for coverage appears only after I refresh the page:

![image](https://user-images.githubusercontent.com/39889/89550048-219b7c80-d7d7-11ea-9713-ce20b46c682d.png)

and apparently it was because I didn't follow the step `2.` you outlined above.  
And pressing on the button indeed highlights coverage but it is a different from what I expected, and the one I get when I just "refresh the page" -- I have highlights in the left "line numbers" column, while pressing on the button highlights the code :

![image](https://user-images.githubusercontent.com/39889/89550776-109f3b00-d7d8-11ea-95ff-16711c1d43bf.png)

apparently I had a `codecov` extension installed/enabled (https://github.com/codecov/browser-extension). But I believe it was deprecated in favor of sourcegraph, so not worth mentioning